packages/sed.rb

```diff
@@ -16,27 +16,14 @@ class Sed < Package
      x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/sed/4.9_x86_64/sed-4.9-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '13f49db4006bd22a894f8c8f870635074438b1b125849ab14220e98205a6b56a',
-     armv7l: '13f49db4006bd22a894f8c8f870635074438b1b125849ab14220e98205a6b56a',
-       i686: 'b601023458812f12c08a5659d5e2033489f8a554fea17b92ea2761b909a3f863',
-     x86_64: 'b0072a360f9a1e921e183f380934b67b5203b6de2104ddfb0441080bd7954bfe'
+    aarch64: '6ad05b86518b574fab280df0788fbb7deffb99effee3b0b38222488dcaad3288',
+     armv7l: '6ad05b86518b574fab280df0788fbb7deffb99effee3b0b38222488dcaad3288',
+       i686: 'ed4650bbd4246cb3f4af2963cb3fc8cacbabcd94794c52ebc07fb45ac52369f5',
+     x86_64: '05c6b559843f27da3226b088ccca87f8eba68e49d994507e28d11e4bc4aff661'
   })
 
-  depends_on 'acl'
+  depends_on 'acl' # R
   depends_on 'glibc' # R
 
-  def self.build
-    system "./configure #{CREW_OPTIONS} --without-selinux"
-    system 'make'
-  end
-
-  def self.check
-    # Remove selinux tests since we're building without it.
-    system "sed -i 's,testsuite/inplace-selinux.sh ,,' Makefile"
-    system 'make', 'check'
-  end
-
-  def self.install
-    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
-  end
+  configure_options '--without-selinux'
```

Comment:
Not entirely confident, but it looks like ChromeOS has SELinux? https://www.chromium.org/chromium-os/developer-library/reference/security/selinux/

Reply:
Historically this has led to build issues...